### DISCUSSION TITLE
静态恶意代码逃逸（第八课） « 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#10] [General]
# 静态恶意代码逃逸（第八课） « 倾旋的博客

倾旋的博客

https://payloads.online/archivers/2020-11-29/1/

**Comment by 0r3ak** [comment]
一直提示：
CryptStringToBinaryA 无法解析外部符号，求解答

**Reply by 0r3ak** [reply]
没事儿了，少了lib
